lib/skate/open_route_service_api/directions_formatter/MBTA/english.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Skate.OpenRouteServiceAPI.DirectionsFormatter.MBTA.English do
+  @moduledoc """
+  Formats Directions from Open Route Service into MBTA English specific Directions Shorthand.
+  """
+
+  @doc """
+  Formats a Open Route Service Direction Instruction Map into
+    - a Instruction, as a `Map` with a single `:instruction` key
+    - or `nil`, which represents discarded instructions
+  """
+  def format(%{"type" => type, "name" => name} = attrs) do
+    case format_instruction(type, name, attrs) do
+      nil -> nil
+      instruction -> %{instruction: instruction}
+    end
+  end
+
+  # Converts ORS Instructions into string form, or `nil`
+  # Ignore noisy instructions
+  defp format_instruction(type, _name, _)
+       when type in [:goal, :depart, :straight, :error],
+       do: nil
+
+  # ORS uses `-` as a value when a direction doesn't have a name
+  # Reject instructions without a name
+  defp format_instruction(_type, "-", _),
+    do: nil
+
+  defp format_instruction(:left, name, _),
+    do: "L - #{name}"
+
+  defp format_instruction(:right, name, _),
+    do: "R - #{name}"
+
+  defp format_instruction(:slight_left, name, _),
+    do: "L - #{name}"
+
+  defp format_instruction(:slight_right, name, _),
+    do: "R - #{name}"
+
+  defp format_instruction(:keep_left, name, _),
+    do: "L - #{name}"
+
+  defp format_instruction(:keep_right, name, _),
+    do: "R - #{name}"
```

Comment:
fix: [Looks like](https://app.asana.com/0/0/1208288732617333/1208592058043925/f) "Keep Left / Right" should be "Keep L / R"

Reply:
https://github.com/mbta/skate/compare/c588d63d5bd2ffa8d4ba61c0d4c68f933552519f..c5c07d6c55b1b10a35234a6fe601a553961ace52